### PR TITLE
fix: Steps with progress - Circle get cut at the bottom

### LINKED ISSUE #30372: Step with progress - Circle get cut at the bottom
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/steps-with-progress-antd4153-forked-evjmx?file=/index.js)

### Steps to reproduce
1. open Step with progress sandbox: https://codesandbox.io/s/8gb6s
2. change the code to: 
```
import React from "react";
import ReactDOM from "react-dom";
import "antd/dist/antd.css";
import "./index.css";
import { Steps } from "antd";

const { Step } = Steps;

ReactDOM.render(
  <Steps current={0} percent={90} style={{ border: "1px solid red" }}>
    <Step title="Finished" />
    <Step title="In Progress" />
    <Step title="Waiting" />
  </Steps>,
  document.getElementById("container")
);
```

### What is expected?
the circle should not get cut at the bottom

### What is actually happening?
the circle get cut at the bottom
image

| Environment | Info |
|---|---|
| antd | 4.15.3 |
| React | 16.14.0 |
| System | macOS Catalina 10.15.7 |
| Browser | Google Chrome 88.0.43 |

---
i found similar issue that has been fixed
https://github.com/ant-design/ant-design/issues/29824

### PR DESCRIPTION
**Pull request by fachrezamuslim**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close #30372

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |Fix Steps with progress - Circle get cut at the bottom|
| 🇨🇳 Chinese |修复Steps进度-在底部切圆|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/steps/style/progress.less [hunk 1]
```diff
@@ -10,6 +10,7 @@
   }
 
   &.@{steps-prefix-cls}-horizontal .@{steps-prefix-cls}-item:first-child {
+    padding-bottom: 4px;
```
Comment (xrkffgg) [inline components/steps/style/progress.less:13]: rtl 这块可以完善一下哈~

Comment (fachreza73) [reply components/steps/style/progress.less:13]: updated

File: components/steps/style/rtl.less [hunk 2]
```diff
@@ -242,3 +242,10 @@
     }
   }
 }
+
+// RTL Steps with progress
+.@{steps-prefix-cls}-rtl.@{steps-prefix-cls}-with-progress.@{steps-prefix-cls}-horizontal.@{steps-prefix-cls}-label-horizontal {
```
Comment (xrkffgg) [inline components/steps/style/rtl.less:247]: 可以参照下上面的写法

Comment (fachreza73) [reply components/steps/style/rtl.less:247]: sorry, i don't understand

Comment (fachreza73) [reply components/steps/style/rtl.less:247]: @xrkffgg could you tell me what should i do?

Comment (xrkffgg) [reply components/steps/style/rtl.less:247]:
```css
// progress
.@{steps-prefix-cls}-with-progress {
  &.@{steps-prefix-cls}-horizontal .@{steps-prefix-cls}-item:first-child {
    .@{steps-prefix-cls}-rtl& {
      padding-right: 4px;
      padding-left: 0;
    }
  }
}
```

**Pull request merged**